src/components/intern-app.html

```diff
@@ -31,24 +31,44 @@
       }
     </style>
     <header>
-      <user-list></user-list>
+      <user-list user-accounts="{{userAccounts}}"></user-list>
       <create-user></create-user>
       <edit-user></edit-user>
     </header>
   </template>
 
   <script type="module">
-    
+
     import { Element as PolymerElement } from '@banno/polymer/polymer-element.js';
-    
+    import { LocalStorage } from '../api/localStorage.js';
+
+    const LocalUsers = new LocalStorage();
+
     class InternApp extends PolymerElement {
+      constructor() {
+        super();
+        this.users = LocalUsers.getAllUsers();
+      }
 
       static get is() { return 'intern-app'; }
-    }
 
+      static get properties() {
+        return {
+          userAccounts: {
+            type: Array,
+            value: () => {
```

Comment:
You can forego the constructor work and just use
```suggestion
            value: () => LocalUsers.getAllUsers()
```